Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM ruby:3.2.0-alpine3.17@sha256:c690d3b45ef65e0098104c272f1736b0595b824908d5639c3a3d17636581a905  AS build
+FROM ruby:3.2.0-alpine3.17@sha256:c690d3b45ef65e0098104c272f1736b0595b824908d5639c3a3d17636581a905 AS build
 
 WORKDIR /app
 
-# Edge repo is necessary for Node 16 and openssl 3
-RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories
 RUN apk update
 RUN apk upgrade --available
-RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs=18.15.0-r0 npm
+RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs npm
```

Comment:
```suggestion
RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs=18.14.2-r0 npm
```

Reply:
This was tested here https://github.com/alphagov/forms-admin/actions/runs/4677839417/jobs/8285819444

Reply:
I suppose we could leave it off